crates/kernel_cmdline/src/utf8.rs

```diff
@@ -426,6 +426,16 @@ mod tests {
     fn test_parameter_quoted() {
         let p = param("foo=\"quoted value\"");
         assert_eq!(p.value(), Some("quoted value"));
+
+        let p = param("foo=\"unclosed quotes");
+        assert_eq!(p.value(), Some("unclosed quotes"));
```

Comment:
I guess this is an answer to my question above, but is it the correct answer?

Reply:
Yeah that's this line - https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/lib/cmdline.c#n256.  We see '=', replace it with '\0', advance one, if it's '"' we advance again past it, effectively stripping it.